R/run-url.R

```diff
@@ -121,7 +121,7 @@ runGist <- function(gist, destdir = NULL, ...) {
 #' @param username GitHub username. If `repo` is of the form
 #'   `"username/repo"`, `username` will be taken from `repo`.
 #' @param ref Desired git reference. Could be a commit, tag, or branch name.
-#'   Defaults to `"master"`.
+#'   Defaults to `"main"`.
```

Comment:
Is this ok?

Reply:
Maybe we could default this to `NA` and try to automatically detect if possible (maybe via usethis)? I feel like straight-up changing it to main is likely to break a lot of existing code

Reply:
Reverting change